ui/desktop/src/api/types.gen.ts

```diff
@@ -22,13 +22,22 @@ export type ConfigResponse = {
     config: {};
 };
 
-export type Content = (TextContent & {
-    type: 'text';
-}) | (ImageContent & {
-    type: 'image';
-}) | (EmbeddedResource & {
-    type: 'resource';
-});
+export type Content = {
+    text: string;
+    type: string;
+} | {
+    data: string;
+    mimeType: string;
+    type: string;
+} | {
+    resource: ResourceContents;
+    type: string;
+} | {
+    annotations?: Annotations;
+    data: string;
+    mimeType: string;
+    type: string;
+};
```

Comment:
this looks like a bit less useful than what we had before

Reply:
Yeah the TS checker passing does give some confidence that this change is OK, but I'll pay some extra attention to this to be sure